pkg/cloudprovider/aws/capacity.go

```diff
@@ -37,33 +37,36 @@ type Capacity struct {
 }
 
 // Create a set of nodes given the constraints.
-func (c *Capacity) Create(ctx context.Context, constraints *cloudprovider.Constraints) ([]cloudprovider.Packing, error) {
-	// 1. Retrieve normalized zones from constraints or all zones that the cluster spans if no zonal constraints are specified
-	zonalSubnetOptions, err := c.vpcProvider.GetZonalSubnets(ctx, constraints, c.spec.Cluster.Name)
+func (c *Capacity) Create(ctx context.Context, cloudProviderConstraints *cloudprovider.Constraints) ([]cloudprovider.Packing, error) {
+	// 1. Create AWS Cloud Provider constraints and apply defaults
+	awsConstraints := NewAWSConstraints(cloudProviderConstraints)
```

Comment:
if you make earlier suggestion, this would just be `... := AWSConstraints(cloudProviderConstraints)`

Reply:
fixed